templates/eu-packager-code-3-parts.156x90.tt.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="45px" height="26px" viewBox="0 0 255 152" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title></title>
+    <desc></desc>
+    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Group">
+            <ellipse id="Oval" stroke="#000000" stroke-width="6" fill="#FFFFFF" cx="127.5" cy="76" rx="124.5" ry="73"></ellipse>
+            <text id="FR" font-family="Arial" font-size="50" font-weight="bold" fill="#000000">
```

Comment:
You could specify `font-family` etc. on the outermost group or via CSS to avoid duplication.